.new_docs/en/build_and_install/gpu.md

```diff
@@ -1,24 +1,26 @@
-[English](../../en/build_and_install/gpu.md) | 简体中文
 
-# GPU部署库编译
+# How to Build GPU Deployment Environment
 
-FastDeploy当前在GPU环境支持Paddle Inference、ONNX Runtime和TensorRT，但同时在Linux&Windows的GPU环境也同时支持CPU硬件，因此编译时也可以同步将CPU的推理后端OpenVINO编译集成
+FastDeploy currently supports Paddle Inference, ONNX Runtime and TensorRT in the GPU environment. It also supports CPU hardware in the GPU environments of Linux & Windows, so developers can compile and integrate the CPU inference backend OpenVINO at the same time.
```

Comment:
FastDeploy supports Paddle Inference, ONNX Runtime and TensorRT in the GPU environment. It should be noted that users also can use CPU in a GPU environment of Linux & Windows, so users can integrate the OpenVINO backend at the same time.